hal/halx86/pc98/halpc98.rc

```diff
@@ -0,0 +1,9 @@
+#define REACTOS_VERSION_DLL
+#if defined(CONFIG_SMP)
+// #define REACTOS_STR_FILE_DESCRIPTION  "NEC PC-98 series Multiprocessor Hardware Abstraction Layer"
```

Comment:
Is a multiprocessor PC-98 NEC actually a thing?

Reply:
Yes. https://chat.reactos.org/reactos/pl/b1psgeyosb8p7fs7y4p97e87qr